magento/pwa-studio
# [FEATURE] Improve className format during development

## Description
When in development finding where a component is located should be as easy as possible. When inspecting an element identifying a component by it's classname is a easy way to do so hence I improved the classname by adding the path to the file where the component is located.

## Related Issue

Closes PWA-1909.

## Acceptance


### Verification Stakeholders



### Specification

## Verification Steps

- pull this pr
- start the watcher
- open the chrome devtools and notice the classnames now point to the actual file in a more explicit format
- close the watcher and start the stage version of the storefront, note that the classes are **not** pointing to the file structure


### Resolved issues:
1. [x] resolves magento/pwa-studio#3294: [FEATURE] Improve className format during development
